src/usage/usage.service.ts

```
@@ -1,29 +1,55 @@
   WidgetKeyStatsDto,
   UsageDataDto,
   DomainStatDto,
 } from '../common/dto/widget-key-usage.dto';
 
+/**
+ * usage_daily에 쓰기 작업을 수행할 실행자(전체 DB 또는 트랜잭션).
+ * 메시지/피드백 저장과 집계 갱신을 하나의 트랜잭션으로 묶을 때 tx를 전달한다.
+ */
+export type UsageDbExecutor =
+  | Database
+  | Parameters<Parameters<Database['transaction']>[0]>[0];
```

ESLint

[error] Replace `|·Database⏎·` with `Database`

(prettier/prettier)